File: daily-coding/coding-bat/string_bits.py
# Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".

def string_bits(str):
  myList = list(str)
  myString = ''
  for x in range(0, len(myList)):
    if x % 2 == 0:
      myString += myList[x]

  return myString

File: daily-coding/coding-bat/java/stringBits.java
// Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".

public String stringBits(String str) {
  String[] myArray = str.split("");
  String newStr = "";
  int length = myArray.length;

  for(int i = 0; i < length;i+=2){
    newStr = newStr + myArray[i];
  }

  return newStr;
}
